### PR TITLE
Implement Task using state/Props

### PR DESCRIPTION
**Pull request by hania-imtisal234**
I have implemented two tasks
1) Take name from user and display the name when button is clicked.
2) Get the current time and checks if it is A.M or P.M and display the related card.

### CODE REVIEW

File: src/Card.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+.logo {
+    height: 158px;
+}
+
+h4 {
+    height: 44px;
+    color: black;
+    font-size: 10px;
+    text-align: center;
+}
```
Comment (usamahmed97) [inline src/Card.css]: Please add EOL.

File: src/Card.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          </a>
+        </div>
+        <div class="Title">
+           <h4>{cardTitle}</h4>
+        </div>
+        
+      </div>
+    );
+  }
+export default Card;
```
Comment (usamahmed97) [inline src/Card.js]: Please add EOL. Also, I think Prettier is not enabled on save. Please check that.

File: src/Card.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import './Card.css'
+
+
+const Card=({img,cardTitle})=> {
+
+    return (
+      <div className="cards">
+        <div class="photo">
+          <a href="#">
+          <img class="logo" src={img}></img>
```
Comment (usamahmed97) [inline src/Card.js]: You may only use  as the self-closing tag.

File: src/Task1.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.task1{
+    width:250px;
+    height:30px;
+    margin-top: 15px;
+    margin-left:10px;
+    margin-right:15px;
+    padding-left:15px;
+}
```
Comment (usamahmed97) [inline src/Task1.css]: Please add EOL.

File: src/Task1.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import './Task1.css'
+import { useState } from 'react';
+
+const Task1= () =>
+{
+    const [input,setinput]= useState(null);
+    const [name,setname]= useState(input)
```
Comment (usamahmed97) [inline src/Task1.js]: Use camel case for this. For example, setName

File: src/Task1.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+       
+        <div className='task1'> 
+          <input type="text" placeholder = "Enter name: " value={name} onChange={setInputName}></input>
+             <button onClick={handleClick}>+</button>
+             <h4>{name}</h4>
+        </div>
+    );
+}
+
+export default Task1;
```
Comment (usamahmed97) [inline src/Task1.js]: EOL

File: src/Task1.js (hunk 5 above)
Comment (usamahmed97) [inline src/Task1.js]: You can use only 1 state for this as the second state defined i.e. name itself is getting the value of input.

File: src/Task1.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+const Task1= () =>
+{
+    const [input,setinput]= useState(null);
+    const [name,setname]= useState(input)
+    const setInputName = (event) =>
+    {
+      setinput(event.target.value);
+    }
+
+    const handleClick=()=>
```
Comment (usamahmed97) [inline src/Task1.js]: No need for this as this can be handled by only one state.

File: src/Task2.css [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    border-color: darkcyan;
+    color:white;
+    margin-top:10px;
+}
+.btn-label
+{
+    color:black;
+    font-size:12px;
+    margin-bottom:-5px;
+}
```
Comment (usamahmed97) [inline src/Task2.css]: EOL.

**Review by usamahmed97 — CHANGES_REQUESTED** [review]
A few changes required.

**Comment by hania-imtisal234** [comment]
I've resolved all the issues

**Review by usamahmed97 — APPROVED** [review]
Looks good. :package:

**Pull request merged**